backend/moonstreamapi/actions.py

```diff
@@ -535,3 +542,27 @@ def apply_moonworm_tasks(
             entries=entries_pack,
             timeout=15,
         )
+
+
+def get_query_by_name(query_name: str, token: uuid.UUID) -> str:
+
+    params = {"type": BUGOUT_RESOURCE_QUERY_RESOLVER, "name": query_name}
+    try:
+        resources: BugoutResources = bc.list_resources(token=token, params=params)
+    except BugoutResponseException as e:
```

Comment:
This exceptions better to describe in api route endpoint in queries.py
Here use something like
```
class ResourceQueryFetchException(Exception):
    """
    ..
    """
```